README.md

```diff
@@ -56,6 +56,8 @@ sum_digest.percentile(30)  # about 0.3
  - `percentile(p)`: return the `p`th percentile. Example: `p=50` is the median.
  - `cdf(x)`: return the CDF the value `x` is at. 
  - `trimmed_mean(p1, p2)`: return the mean of data set without the values below and above the `p1` and `p2` percentile respectively. 
+ - `serialize()`: return a Python dictionary of the TDigest and internal Centroid values.
```

Comment:
is this the right name? this makes me think it stores it on dict. What about `to_dict()` and `from_dict(d)` as names?

Reply:
Yes, I agree! Ok I will change it.

Reply:
need to update these names too